lib/uvobot/notifications/discourse_notifier.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'notifier'
+require_relative '../uvo_scraper'
+
+module Uvobot
+  module Notifications
+    class DiscourseNotifier < Notifier
+      def initialize(discourse_client, category, scraper)
+        @client = discourse_client
+        @category = category
+        @scraper = scraper
+      end
+
+      def no_announcements_found
+        # noop
+      end
+
+      def new_issue_not_published
+        # noop
+      end
+
+      def matching_announcements_found(_page_info, announcements)
+        announcements.each do |a|
+          topic = announcement_to_topic(a)
+          @client.create_topic(
+            title: topic[:title],
+            raw: topic[:body],
+            category: @category
+          )
+        end
+      end
+
+      private
+
+      def announcement_to_topic(announcement)
+        detail = @scraper.get_announcement_detail(announcement[:link][:href])
+
+        {
```

Comment:
Je to tam ponate tak, ze metoda detail_message da namiesto ceny spravu o nepodarenej extrakcii. A link na detail/zdroj obstaravania, maju hned pod tym.

To fakt chces tvorit zvlast topic s oznamom ze sa nepodarilo vytiahnut cenu pre kazde obstaravanie co ma iny format?